source/user-manual/api/getting-started.rst

```diff
@@ -24,78 +24,224 @@ The API starts at boot time. To control or check the **wazuh-api** service, use
+
+.. thumbnail:: ../../images/api/api-ui-overview.png
+    :align: center
+    :width: 100%
+
+The endpoints can be used both in this interface and with the command-line tool *curl*, so in some examples we will use the two options.
+
+.. _api_log_in:
+
+Logging into the API
+--------------------
+
+Wazuh API endpoints require authentication in order to be used. Therefore, all calls must include a JSON Web Token. JWT is an open standard (RFC 7519) that defines a compact and self-contained way for securely transmitting information between parties as a JSON object.
+
+Logging into the UI
+^^^^^^^^^^^^^^^^^^^^^^
+
+1. Access the API interface at <protocol>://<host>:<port>/ui. The default path is ``https://localhost:55000/ui``.
+2. Click on the green *Authorize* button.
+3. Enter the username and password inside the section *basicAuth  (http, Basic)*. By default, the user is ``wazuh`` and the password is ``wazuh``.
+4. Click on the green *Authorize* button under the password field and close this window.
+5. Expand the endpoint ``GET /security/user/authenticate`` and click on the *Try it out* button.
+6. Click on the *Execute* button. A result like the following will appear. Copy the value that appeared in ``<YOUR_JWT_TOKEN>``, without the quotes.
+
+    .. code-block:: json
+        :class: output
+
+        {
+          "token": "<YOUR_JWT_TOKEN>"
+        }
+
+7. Click again on the Authorize button.
+8. Paste the value of the copied token into the *jwt  (http, Bearer)* field.
+9. Click the Authorize button inside. After this, you can use any API endpoint allowed for the role to which the user belongs.
+
+Logging in with cURL
+^^^^^^^^^^^^^^^^^^^^
+
+1. Use the cURL command to get the JWT token. Replace <user> and <password> with yours. By default, the user is ``wazuh`` and the password is ``wazuh``.  If ``SSL`` (https) is enabled in the API and it is using the default self-signed certificates, it will be necessary to add the parameter ``-k`` as in the example above. Otherwise, it is not necessary to include ``-k``.
+
+    .. code-block:: console
+
+        # curl -u <user>:<password> -X GET "https://localhost:55000/v4/security/user/authenticate"
+
+    You will get a result similar to the following. Copy the token that you will find in ``<YOUR_JWT_TOKEN>`` without the quotes.
+
+    .. code-block:: none
+        :class: output
+
+        {"token": "<YOUR_JWT_TOKEN>"}
+
+2. Run any endpoint following this structure. Replace <endpoint> and <YOUR_JWT_TOKEN> with your values.
```

Comment:
this shouldn't be step 2 but an actual example of how to use the token once the user has it.